csp/contrib/rate_limiting.py

```diff
@@ -23,7 +23,8 @@ def get_policy_parts(self, request: HttpRequest, response: HttpResponseBase, rep
         if policy is None:
             return policy_parts
 
-        remove_report = random.randint(0, 99) >= policy.get("REPORT_PERCENTAGE", 100)
+        # `random.random` returns a value in the range [0.0, 1.0) so all values will be < 100.0.
```

Comment:
```suggestion
        # `random.random` returns a value in the range [0.0, 1.0] so all values will be < 100.0.
```

Reply:
That was actually intentional to denote that 1.0 isn't included in the range that `random.random()` can return.